### ISSUE TITLE
Possible roll into the Type Cleanup add-on?

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/TypeExport#2]
This tool has little interface except the GTK file selector dialog.

Maybe its features could be rolled into an upgraded Type Cleanup? Maybe Paul could be invited to discuss the possibility?

Maybe add buttons in the leftside 'Types' panel ?  At the TOP (to give visual separation from the Help & Remove buttons) have buttons for Copy (to OS clipboard), Import and Export.
![image](https://user-images.githubusercontent.com/69127217/197810187-b8c98533-b62a-487b-84b7-9cdfcc71919c.png)

(The added Copy functionality is for easy sharing to the Discourse forum or by eMail.)

**Comment by cdhorn** [comment]
I did consider reworking that and folding in there, but then it wouldn't just be a Cleanup tool anymore. Could get renamed to Types Management Tool and then add ability to enter new ones as well perhaps. Not sure is worth spending that much time on though at the moment.

I wanted to use yaml for the export, except I recall asking Paul once and the WindowsAIO wasn't packaged with it at the time if I recall.

**Comment by emyoulation** [comment]
Yeah, it's not reasonable to invest much time considering how infrequently the ini related feature will see use.

That's one of the reasons I thought that folding it into the Type Cleanup tool would be to best path. Any other direction implies too much GUI work. And GUI tempts users to play where they shouldn't.

An 'Add new' would need to much interface to provide casual users enough information to make good decisions.

An over-padded 'cleanup' tool is better than an under-featured 'management' tool tool.
